README.md

```diff
@@ -116,3 +116,11 @@ $ curl --silent https://mgdm.net | htmlq --pretty '#posts'
     </li>
 [...]
 ```
+
+### Syntax highlighting with [`bat`](https://github.com/sharkdp/bat)
+
+```console
+$ curl -s example.com | htmlq 'body' | bat -l html
```

Comment:
Just saw PR #11, are you interested to align with it? It is like:

```sh
curl --silent example.com | htmlq 'body' | bat --language html
```

Reply:
Done.